typescript/sdk/src/deploy/core/HyperlaneCoreDeployer.ts

```diff
@@ -42,77 +38,62 @@ export class HyperlaneCoreDeployer<
     this.startingBlockNumbers = objMap(configMap, () => undefined);
   }
 
-  // override return type for inboxes shape derived from chain
-  async deploy(
-    partialDeployment?: Partial<CoreContractsMap<Chain>>,
-  ): Promise<CoreContractsMap<Chain>> {
-    return super.deploy(partialDeployment) as Promise<CoreContractsMap<Chain>>;
-  }
-
-  async deployOutbox<LocalChain extends Chain>(
+  async deployMailbox<LocalChain extends Chain>(
     chain: LocalChain,
-    config: ValidatorManagerConfig,
+    moduleAddress: types.Address,
     ubcAddress: types.Address,
-  ): Promise<OutboxContracts> {
+  ): Promise<ProxiedContract<Mailbox, BeaconProxyAddresses>> {
     const domain = chainMetadata[chain].id;
-    const outboxValidatorManager = await this.deployContract(
-      chain,
-      'outboxValidatorManager',
-      [domain, config.validators, config.threshold],
-    );
 
-    const outbox = await this.deployProxiedContract(
+    const mailbox = await this.deployProxiedContract(
       chain,
-      'outbox',
+      'mailbox',
       [domain],
       ubcAddress,
-      [outboxValidatorManager.address],
+      [moduleAddress],
     );
-    return { outbox, outboxValidatorManager };
+    return mailbox;
   }
 
-  async deployInbox<Local extends Chain>(
-    localChain: Local,
-    remoteChain: Remotes<Chain, Local>,
-    config: ValidatorManagerConfig,
-    ubcAddress: types.Address,
-    duplicate?: ProxiedContract<Inbox, BeaconProxyAddresses>,
-  ): Promise<InboxContracts> {
-    const localDomain = chainMetadata[localChain].id;
-    const remoteDomain = chainMetadata[remoteChain].id;
-    const inboxValidatorManager = await this.deployContract(
-      localChain,
-      'inboxValidatorManager',
-      [remoteDomain, config.validators, config.threshold],
-    );
+  async deployMultisigIsm<LocalChain extends Chain>(
+    chain: LocalChain,
+  ): Promise<MultisigIsm> {
+    const module = await this.deployContract(chain, 'multisigIsm', []);
+    const configChains = Object.keys(this.configMap) as Chain[];
+    const remotes = this.multiProvider
+      .intersect(configChains, false)
+      .multiProvider.remoteChains(chain);
+    await super.runIfOwner(chain, module, async () => {
+      // TODO: Remove extraneous validators
+      for (const remote of remotes) {
+        const moduleConfig = this.configMap[remote].multisigIsm;
```

Comment:
also wondering how you'd feel about `s/module/ism/g` throughout here

Reply:
Good catch, missed this.